css/style.css

```
@@ -62,6 +194,38 @@
   background-repeat: no-repeat;
 }
 
+.subscription-title {
+  font-size: 30px;
+  line-height: 36px;
+  font-weight: 700;
+  color: #ffffff;
+  text-transform: uppercase;
+}
+
+.subscription-description {
+  color: #ffffff;
+}
+
+.subscription-field {
+  font-size: 18px;
+  background-color: #f2f2f2;
+}
+
+.subscription-field::placeholder {
+  color: #000000;
+}
+
+.subscription-button {
+  font-size: 22px;
```

```suggestion
  font-size: 20px;
```
Кнопке нужно указать шрифт напрямую, а то возьмет от браузера